bin/minasu-client.py

```diff
@@ -1,31 +1,28 @@
 #!/usr/bin/env python3
 
-#-----------------
+# -----------------
```

Comment:
I don't actually like the space between the # and the -. 

#------
vs.
# ------

Linter hate :hear_no_evil: